packages/core/src/Tab/Tab.tsx

```diff
@@ -0,0 +1,163 @@
+import * as Tabs from '@radix-ui/react-tabs'
+import React from 'react'
+import { ResponsiveValue } from 'styled-system'
+import { Box } from '../Box/Box'
+import { IconComponent } from '../Chip/ChipContent/ChipContent'
+import { ChoiceChipProps } from '../Chip/ChoiceChip/ChoiceChip'
+import { Grid } from '../Grid/Grid'
+import { TabList } from './Tab.styled'
+import { TabButton } from './TabButton'
+import { TabChip } from './TabChip'
+import { TabRadio } from './TabRadio'
+
+/**
+ * Represents the data for a single tab in the tab component.
+ * @public
+ */ export type TabDataProps = {
+  id: string
+  icon?: IconComponent
+  text: string
+  children: React.ReactNode
+  onTabSelect: () => void
+}
+/**
+ * Represents the props for a tab component.
+ * @public
+ */
+export type TabProps = {
+  ariaLabel?: string
+  orientation?: 'horizontal' | 'vertical'
+  size?: ResponsiveValue<'sm' | 'md'>
+  tabsData: TabDataProps[]
+  variant?: string
+  border?: boolean
+  tabGap?: number
+  type: 'chip' | 'radio' | 'button'
+  isTransparent?: boolean
+  buttonChipProps?: ChoiceChipProps
+  dynamicTabWidth?: boolean
+  defaultValue?: string
+}
+export type MappedTabProps = {
+  tab: TabDataProps
+  buttonChipProps?: ChoiceChipProps
+  isActive?: string
+  onTabSelect?: () => void
+}
+export const Tab = ({
+  tabsData,
+  orientation = 'horizontal',
+  size = 'md',
+  type = 'button',
+  border = false,
+  tabGap,
+  isTransparent = false,
+  dynamicTabWidth = false,
+  ariaLabel = 'PCLN Tabs',
```

Comment:
not sure the default aria label should contain PCLN

Reply:
Do you think keeping it as just 'Tabs' would be fine?